setup.cfg

```diff
@@ -10,6 +10,7 @@ show-source = True
 statistics = False
 doctests = True
 inline-quotes = double
+max-line-length = 88 # The black formatter expects a line length of 88 characters
```

Comment:
I found this useful when developing locally. I can remove this if you want.

Reply:
I think we should do either 79 (according to PEP8) or 119 (According to IntelliJ/other IDEs). I'm happy with either!

Reply:
If we want to not use 88 , then we need to change our black config somehow. I'm going to leave this as is (this change isn't in the repo in this state anymore), and we can change it in another PR if you want.